tuxemon/core/config.py

```diff
@@ -115,7 +119,18 @@ def __init__(self, config_path=None):
         # input config (None means use default for the platform)
         self.gamepad_deadzone = .25
         self.gamepad_button_map = None
-        self.keyboard_button_map = None
+        self.keyboard_button_map = {
```

Comment:
this is a great start.  it looks correct, but maybe we could make this a bit more DRY by refactoring into a single method?  And it also looks like it is going to change the defaults in `PygameKeyboardInput.default_input_map`.  hope you don't mind this suggestion, but maybe we could do something like
```python
class TuxemonConfig(object):
    def __init__(self, config_path=None):
      ...
      self.keyboard_button_map = get_custom_pygame_keybaord_controls(cfg)
      ...

def get_custom_pygame_keybaord_controls(cfg):
    import pygame.locals
    from tuxemon.core.platform.platform_pygame.events import PygameKeyboardInput

    custom_controls = PygameKeyboardInput.default_input_map.copy()
    for key, value in cfg.items("controls"):
        key = key.lower()
        value = value.upper()
        pygame_value = getattr(pygame.locals, "K_" + key, None)
        button_value = getattr(buttons, value, None)
        if pygame_value is not None and button_value is not None:
            custom_controls[pygame_value] = button_value
    return custom_controls
```

Reply:
Yeah, this idea looks great. I'll definitely get started on this.